examples/example-probes/src/global_var/mod.rs

```diff
@@ -0,0 +1,11 @@
+use cty::*;
+
+// This is where you should define the types shared by the kernel and user
```

Comment:
I think these variables should be declared on the crate level. Since it's the correct compilation unit for both userland and kernelspace, they should at least able to access the typing of what's declared here.

By doing this, we could come up with a proc macro, and/or generic `Atomic<T>` that wraps these variables in the userland with safe accessors, and leaves them alone in BPF.

Reply:
Oh that's a good idea.
I didn't come up with this since I just considered collecting info of global variables in probes by ELF symbol table or BTF info.
Your suggestion shows me alternative designs. Cool.
Thanks,